### ISSUE TITLE
Please add dynamic adapter loader by config

### ISSUE THREAD

**Issue by Kachit** [phalcon/cphalcon#11001]
For example:

``` php
$config = [
'adapter' => '{adapterName}',
'host' => 'host',
'user' => 'user',
//etc...
];

/* @var Phalcon\Db\Adapter\Pdo\{AdapterName} $dbAdapter*/
$dbAdapter = Phalcon\Db\Adapter\Pdo\Factory::load($config);
```

This feature actual for phalcon components with multiple adapters (db, sessions, cache etc)

**Comment by mapirelli** [comment]
i like it

**Comment by Jurigag** [comment]
@Kachit @mapirelli @sergeyklay 

what `$dbAdapter = Phalcon\Db\Adapter\Pdo\Factory::load($config);` should return ? `Phalcon\Db\Adapter\Pdo` or `Phalcon\Db\AdapterInterface` ?

**Comment by Jurigag** [comment]
Implemented in 3.2.x
